lumicks/pylake/fdcurve.py

```diff
@@ -99,6 +132,24 @@ def d(self):
             self._distance_cache = getattr(self, f"distance{self._primary_distance_channel}")
         return self._distance_cache
 
+    def get_fd_range(self, force_min=0, force_max=np.inf, distance_min=0, distance_max=np.inf):
```

Comment:
Sure thing.

I totally agree about the trim. Especially since we don't raise on subtracting a force bigger than the minimum force. Was an oversight on my part, so good catch! Thank you.